schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -859,6 +859,9 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Compute.json#/resourceDefinitions/virtualMachineScaleSets" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Compute.Extensions.json#/resourceDefinitions/virtualMachines_extensions" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.Compute.Extensions.json#/resourceDefinitions/virtualMachineScaleSets_extensions" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-03-01/Microsoft.Compute.Galleries.json#/resourceDefinitions/galleries" },
```

Comment:
>Microsoft.Compute.Galleries.json [](start = 84, length = 32)

PR looks good. one question: we don't have 2019-03-01 version of Microsoft.Compute.json today?

Reply:
It is the latest api version for azure compute. People are gradually adding support of this api version.